rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java

```diff
@@ -64,52 +61,22 @@ public ListJobCommand(String... filtering) {
     @Override
     public void execute(ApplicationContext currentContext) throws CLIException {
         try {
-            if (isIteractiveMode(currentContext)) {
-                int numOfJobsPerPage = getNumOfJobsPerPage(currentContext);
-                int numOfJobs = getNumberOfJobs(beginIndex, offset, currentContext);
-                if (numOfJobs < numOfJobsPerPage) {
-                    printJobsList(beginIndex, numOfJobs, currentContext);
-                } else {
-                    int pages = (numOfJobs / numOfJobsPerPage) + ((numOfJobs % numOfJobsPerPage) == 0 ? 0 : 1);
-                    int iterCount = 0;
-                    boolean hasQuit = false;
-                    // all pages except the last one
-                    for (; iterCount < pages - 1; iterCount++) {
-                        printJobsList(beginIndex + (iterCount * numOfJobsPerPage), numOfJobsPerPage, currentContext);
-                        String response = readLine(currentContext,
-                                                   "Page(%d/%d). Press ENTER to continue. 'q' to quit. ",
-                                                   (iterCount + 1),
-                                                   pages);
-                        if ("q".equalsIgnoreCase(response)) {
-                            hasQuit = true;
-                            break;
-                        }
-                    }
-                    if (!hasQuit) {
-                        // last page
-                        printJobsList(beginIndex + (iterCount * numOfJobsPerPage),
-                                      numOfJobs - (iterCount * numOfJobsPerPage),
-                                      currentContext);
-                    }
-                }
-            } else {
-                printJobsList(beginIndex, (offsetFlag) ? offset : RestConstants.DFLT_PAGINATION_SIZE, currentContext);
-            }
```

Comment:
@paraita we tested it for the two modes. In all cases, it renders the list of jobs in one single page, and not only the last 50.
./proactive-client -lj is equivalent to >help() >listjobs()
./proactive-client -lj limit=67 is equivalent to >help() >listjobs(67)
./proactive-client -lj from=1 limit=51 is equivalent to >help() >listjobs(1,51)